build-system/server.js

```diff
@@ -404,6 +404,21 @@ app.use('/examples/amp-fresh.amp.(min.|max.)?html', function(req, res, next) {
     next();
 });
 
+
+app.use('/impression-proxy/', function(req, res) {
+  assertCors(req, res, ['GET']);
+  // Fake response with the following optional fields:
+  // location: The Url the that server would have sent redirect to w/o ALP
+  // tracking_url: URL that should be requested to track click
+  // gclid: The conversion tracking value
+  const body = {
+    'location': 'localhost:8000/examples/#gclid=1234?gclid=RANDOM&r=1&url=SOURCE_URL',
```

Comment:
I honestly don't know how to write a real querystring. Can you give me an example or lead to a certain doc i can take a look at?